.editorconfig

```diff
@@ -3,3 +3,6 @@ root = true
 [*.json]
 indent_size = 2
 indent_style = spaces
+insert_final_newline = true
+# end_of_line = lf
```

Comment:
Both should be fine :) I'd even go as far as moving insert_final_newline, end_of_line and charset to [*].